tests/dialects/test_snowflake.py

```diff
@@ -2289,3 +2289,45 @@ def test_window_function_arg(self):
         self.assertEqual(ast.sql("snowflake"), query)
         self.assertEqual(len(list(ast.find_all(exp.Column))), 1)
         self.assertEqual(window.this.sql("snowflake"), "db.schema.FUNC(a)")
+
+    def test_alias_expansion(self):
```

Comment:
Let's not roll a custom test method for this, we have a fixture file called `qualify_columns.sql` that contains similar tests.